components/posts/NftPost.vue

```diff
@@ -1,6 +1,25 @@
 <template>
-  <div class="flex flex-col justify-center items-center h-60 border-gray-300 border-2 m-4">
-    <p>TODO!</p>
+  <div class="flex flex-col justify-left items-left h-60 border-gray-300 bg-white border-2 mx-4 my-2 rounded">
+    <div class="border-gray-300 border-2 p-4 height-300 m-4 flex justify-center items-center rounded">
+      <p>image here</p>
+    </div>
+    <div class="flex flex-row grid-colls-2 gap-3 mx-4 my-2">
+      <div>
+        <p>pfp</p>
+      </div>
+      <div class="font-bold text-lg">
+        <p>username here</p>
+      </div>
+    </div>
+    <div class="mx-4 my-2">
+      <p>NFT description here...</p>
+    </div>
+    <div class="mx-4 my-2 flex flex-wrap items-center gap-2">
+      <p class="bg-slate-600 text-white px-2 py-1 rounded">NFT</p>
+      <p class="bg-slate-600 text-white px-2 py-1 rounded">tags</p>
+      <p class="bg-slate-600 text-white px-2 py-1 rounded">will be here</p>
+      <p class="underline text-gray-300">Show more</p>
+    </div>
   </div>
 </template>
```

Comment:
You could also mock some data in parent component and pass them by the props (image, tags, owner, etc).